src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -8,7 +8,7 @@ import { RenderQueue } from './Platform/Circuits/RenderQueue';
 import { ConsoleLogger } from './Platform/Logging/Loggers';
 import { LogLevel, Logger } from './Platform/Logging/Logger';
 import { CircuitDescriptor } from './Platform/Circuits/CircuitManager';
-import { setEventDispatcher } from './Rendering/RendererEventDispatcher';
+import { setEventDispatcher } from './Rendering/Events/EventDispatcher';
```

Comment:
You'll see quite a bit of renaming and file moving in this PR for the TypeScript "events" related code. This is to clean up all kinds of cruft that had accumulated in this area.